src/main/java/orm/PersistenceContext.java

```diff
@@ -0,0 +1,16 @@
+package orm;
+
+public interface PersistenceContext {
+
+    <T> T getEntity(Class<T> entityClazz, Object id);
+
+    <T> T addEntity(T entity);
+
+    boolean containsEntity(EntityKey key);
+
+    void updateEntity(Object entity);
+
+    void removeEntity(Object entity);
+
+    void clear();
```

Comment:
```suggestion
    void updateEntity(Object entity);

    void removeEntity(Object entity);
```
사용하지 않는 메소드인 것 같네요 :)

Reply:
넵 하이버네이트 구현체를 보다가 필요해보인다고 생각해서 넣어놨는데 실제로 필요하진 않았네요